src/open-id-connect.ts

```diff
@@ -26,6 +26,8 @@ export default class OpenIdConnect {
   }
 
   public async login(args: any = {}): Promise<void> {
+    const instruction = this.router.currentInstruction;
+    const redirectUrl = instruction.queryParams.loginRedirectRoute || this.getInstructionUrl(instruction);
```

Comment:
Redirect URL is unused. What is its purpose here?

Reply:
It had in https://github.com/aurelia-contrib/aurelia-open-id-connect/commit/072c025b5d33541b5e0a2a943bc1f220fd20708f until you merge other pull request and broke it. 
See the history of [open-id-connect.ts](https://github.com/aurelia-contrib/aurelia-open-id-connect/commits/master/src/open-id-connect.ts).

Reply:
Good to know. Thank you for following up on that.